sdk/typescript/src/providers/json-rpc-provider.ts

```diff
@@ -161,6 +167,26 @@ export class JsonRpcProvider {
 		return requestSuiFromFaucet(this.connection.faucet, recipient, httpHeaders);
 	}
 
+	async requestSuiFromFaucetV1(
```

Comment:
It's maybe confusing to the user which one to use given there's an unversioned one and V1. Can you rename the other one V0 or something to make it obvious which to use?

Reply:
I don't want to rename it since that would be a breaking SDK change

Reply:
IMO that's probably fine, it's a pretty small API change and every `minor` release can actually introduce breaking changes because we're pre-major.

Reply:
Just make the changeset explain how to update and make sure docs are updated.